README.md

```diff
@@ -1129,6 +1129,8 @@ first([
 //! "nope"
 ```
 
+When one Future resolves, the other gets cancelled automatically.
```

Comment:
Shall we use the term *settles* (instead of *resolves*) to mean *to resolve __or__ reject*?